client/my-sites/site-settings/site-tools/index.jsx

```diff
@@ -94,7 +94,7 @@ class SiteTools extends Component {
 		return (
 			<div className="site-tools">
 				<QueryRewindState siteId={ siteId } />
-				<SettingsSectionHeader title={ translate( 'Site Tools' ) } />
+				<SettingsSectionHeader title={ translate( 'Site tools' ) } />
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.